src/exception.rs

```diff
@@ -71,7 +71,12 @@ core::arch::global_asm!(
 ///
 pub fn handle_exception_sync(ctx: &mut TrapFrame) -> AxResult<AxVCpuExitReason> {
     match exception_class() {
-        Some(ESR_EL2::EC::Value::DataAbortLowerEL) => handle_data_abort(ctx),
+        Some(ESR_EL2::EC::Value::DataAbortLowerEL) => {
+            let elr = ctx.exception_pc();
```

Comment:
Why not put this in `handle_data_abort`?

Reply:
I don't think it is necessary. 

For some exceptions,(e.g. DataAbortLowerEL, SMC64) we need to manually modify the elr. 

it is more intuitive to put these logics together.

Reply:
OK